### ISSUE TITLE
Vulkan performance Worse than OpenGL in Android

### ISSUE THREAD

**Issue by KaelMa** [google/filament#4225]
**Describe the bug**
I tried android sample-gltf-viewer app with a 10M glb, 
vulkan-backend fps is 43, while opengl-backend fps is 60.

**To Reproduce**
just change backend 

**Expected behavior**
vulkan performance better than opengl

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [ Android]
 - Backend: [OpenGL/Vulkan]

**Smartphone (please complete the following information):**
 - Device: [Pixel 1, Huawei Mate]

**Comment by prideout** [comment]
Which device are you using?

**Comment by KaelMa** [comment]
> I don't have a Pixel 1 or Huawei Mate but I can do some performance analysis with other devices.
> 
> You said your asset is 10M, do you see similar behavior with a glb that could be attached to the issue?

This glb link: https://drive.google.com/file/d/1xwKpoiMxDfhLc8lMpknbA7w-VZRkrgvg/view?usp=sharing
Ω
![image](https://user-images.githubusercontent.com/8552840/124060750-44262c00-da60-11eb-8b50-925f26c3c4fc.png)
I upload this glb to google 

I have tried other glbs, this problem can still reproduce. 

I used to think the problem is vulkan validation layers, then I tried released app without validation layers, but it still not worked.

My profiling tool is a game profile tool , which tracks SurfaceFlinger fence, so I thick it's fps result is relatively trustful.
